src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -13,20 +13,22 @@ public Figure getRandomFigure() {
         ColorSupplier randomColor = new ColorSupplier();
         Random random = new Random();
         int randomNumber = random.nextInt(4);
+        int randomNumber1 = random.nextInt(100);
+        int randomNumber2 = random.nextInt(100);
+        int randomNumber3 = random.nextInt(100);
         switch (randomNumber) {
             case (0):
-                return new Circle(randomColor.getRandomColor(), random.nextInt(100));
+                return new Circle(randomColor.getRandomColor(), randomNumber1);
```

Comment:
![image](https://user-images.githubusercontent.com/58442482/130079238-1c24acd8-7e24-45d7-81ad-bd738c8070e8.png)
create these for all figures and use them in switch case